### ISSUE TITLE
Add more hooman demos!

### ISSUE THREAD

**Issue by Abdur-rahmaanJ** [Abdur-rahmaanJ/hooman#43]
Add more demos to the hooman/demos folder!

**Comment by tuhinmallick** [comment]
Is this issue still available then I would like to give it a try.

**Comment by Abdur-rahmaanJ** [comment]
@tuhinmallick Yes, it's perpetually open! Just find something to do with Hooman and PR a demo.

**Comment by SahajaThota** [comment]
Hi @Abdur-rahmaanJ,
Is this issue still available? Can I work on it.

Thanks.

**Comment by Abdur-rahmaanJ** [comment]
@SahajaThota This is an always open issue ^^, especially to help out new people starting out!

**Comment by SahajaThota** [comment]
Hi @Abdur-rahmaanJ,
I have added demo_bouncing_ball.py to the hooman/demos folder. Please let me know if anything needs to be changed.

Thanks.

**Comment by SahajaThota** [comment]
Hi @Abdur-rahmaanJ,
The library is very interesting and I enjoyed working on rotatingsquare.py. I have added it to the hooman/demos folder. Can you please review and merge it. PR-https://github.com/Abdur-rahmaanJ/hooman/pull/59

Thanks.
Sahaja

**Comment by Jagathi-kota** [comment]
Hi @Abdur-rahmaanJ,
 I have added shootingstar.py it to the hoomanfolder. Can you please review and merge it. PR-https://github.com/Abdur-rahmaanJ/hooman/pull/61

Thanks.

**Comment by chandana1221** [comment]
Hi @Abdur-rahmaanJ,
I have added onclickcolor.py to the hooman folder. Can you please review and merge it. PR-https://github.com/Abdur-rahmaanJ/hooman/pull/62

Thanks,

**Comment by bharathchittamuru7** [comment]
Hi @Abdur-rahmaanJ,
I have added randomlymovingsquares.py to the hooman folder. Can you please review and merge it. PR-https://github.com/Abdur-rahmaanJ/hooman/pull/63.

Thanks,
Bharath.

**Comment by Jattulu** [comment]
Hi @Abdur-rahmaanJ,
I have added collectingballs.py to the hooman/demon folder. Can you please review and merge it. PR-https://github.com/Abdur-rahmaanJ/hooman/pull/69

**Comment by atallur2001** [comment]
Hi @Abdur-rahmaanJ,
I loved working on dodgefallingobj.py and added it to hooman/demon folder. Can you please review and merge it. PR-https://github.com/Abdur-rahmaanJ/hooman/pull/70
Thanks,
Abhinaya.

**Comment by anvithayerneni** [comment]
Hello @Abdur-rahmaanJ, I hope you will like my pong game.
I have added pong_game.py to the hooman/demo folder. I request that you have a look at it and merge it.
Thanks and Regards,
Anvitha Yerneni
#76

**Comment by Manan50** [comment]
Hello @Abdur-rahmaanJ Can I also work on this if available?

**Comment by Mohansai48** [comment]
Hi @Abdur-rahmaanJ ,
I have added mazerunner.py to the hooman/demos folder. Can you please review and merge it. 
PR- https://github.com/Abdur-rahmaanJ/hooman/pull/93

**Comment by Abdur-rahmaanJ** [comment]
@Manan50 Yes, sure, always available, just send a Pull Request!

**Comment by psmanam** [comment]
Hi 
I have added gun_shoot_game1.py. The PR link is below.
[PR_94](https://github.com/Abdur-rahmaanJ/hooman/issues/43)
